### PR TITLE
more realistic Florianne

### PR DESCRIPTION
**Pull request by florianne1212**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def notify_channels():
 	slots = ["12h15", "12h30", "12h45", "13h00", "13h15", "13h30", "13h45"];
 	for channel in channel_list:
 		print(get_current_time() + ' notifying: ' + channel.name)
-		client.loop.create_task(channel.send('Le foodtruck est ouvert, ' + numpy.random.choice(slots, 1, p=[0.1, 0.1, 0.1, 0.2, 0.2, 0.2, 0.1])[0] + ' vite !!! @everyone'))
+		client.loop.create_task(channel.send('Le foodtruck est ouvert, ' + numpy.random.choice(slots, 1, p=[0.3, 0.2, 0.1, 0.1, 0.1, 0.1, 0.1])[0] + ' vite !!! @everyone'))
```
Comment (avan-pra) [inline main.py]: Hours are too unrealistic, we need more 13h15

Comment (florianne1212) [reply main.py]: we can do : [0.2, 0.2, 0.1, 0.2, 0.1, 0.1, 0.1]

Comment (florianne1212) [reply main.py]: 12h30, 12h45 and 13h15 have the same probability

Comment (avan-pra) [reply main.py]: no one eat's at 12h15, we're not baby's

Comment (florianne1212) [reply main.py]: Sorry I meant [0.1, 0.2, 0.2, 0.1, 0.2, 0.1, 0.1]

**Pull request merged**